### DISCUSSION TITLE
exercise name problem

### DISCUSSION THREAD

**Discussion by clafalco** [wger-project/wger#1224] [General]
Hi! in the app
the id appears in the app, instead of displaying the name of the exercise.
![Screenshot_2023-01-11-12-24-02-606_de wger flutter](https://user-images.githubusercontent.com/122402237/211794619-21372c1e-5d6d-4f7e-ad48-bbfef78b3d27.jpg)

**Comment by rolandgeider** [comment]
Hi! thanks for reporting this, there is already [a PR for this](https://github.com/wger-project/flutter/pull/300), we just need to finish some details and merge it
